ProfileManager.js

```diff
@@ -745,6 +747,8 @@ export default class ProfileManager {
   }
 
   async _getAgentRecord({profileId}) {
+    // FIXME is this check needed on private methods?
```

Comment:
I would think not.

Reply:
done https://github.com/digitalbazaar/bedrock-web-profile-manager/pull/26/commits/bcb23b615c5085b0c6a4084934bd73c5ee66b735